testing/test_events.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import eth_utils
+from ethindex import pgimport
+
+
+def make_address(i: int):
+    return eth_utils.to_checksum_address("0x{:040X}".format(i))
+
+
+@pytest.fixture
+def add_some_tranfer_events(testenv):
+    for i, contract in enumerate(testenv.contracts):
+        contract.functions.makeTransfer(
+            make_address(i), make_address(i + 1), i
+        ).transact()
+
+
+def test_get_events(testenv, add_some_tranfer_events):
+    events = pgimport.get_events(testenv.web3, testenv.topic_index, 0, "latest")
+    print("EVENTS:", len(events), events)
```

Comment:
Please no print in tests

Reply:
pytest captures them and only shows them when a test fails. IMO this helps tremendously when debugging a failing test. 

I fail to see a reason to forbid them in tests. Care to elaborate?

Reply:
Only glutter, so if there are not shown normally im fine.